kedify-agent/templates/agent-rbac.yaml

```diff
@@ -0,0 +1,449 @@
+{{- if .Values.agent.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: kedify-leader-election
+  namespace: {{ .Release.Namespace }}
+  labels:
+    control-plane: kedify-agent
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
luckily we should be able to remove this per https://github.com/kedify/agent/pull/242

Reply:
ok, I am removing this. Once the agent is failing on this, we can either add it back or fix the agent. iirc, there were some issues w/ the shared informers that required the "global" verbs. I mean those for CMs called:

  - kedify-state
  - kedify-http-addon-state

were not enough

Reply:
or have I fixed this? I forgot, lol

Reply:
should be good now :)